components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -251,6 +251,14 @@ class EditPostModal extends React.PureComponent {
         this.props.actions.hideEditPostModal();
     }
 
+    handleCheckForChangesHide = (doRefocus = true) => {
```

Comment:
This is nitpicking but Why even bother adding the `doRefocus` here? it is just called once and you can just call `this.handleHide()`